app/controllers/supplejack_api/stories/moderations_controller.rb

```diff
@@ -7,9 +7,25 @@ class ModerationsController < SupplejackApplicationController
       before_action :authenticate_admin!
 
       def index
-        @user_sets = UserSet.all_public_sets
-        render json: @user_sets, each_serializer: StoriesModerationSerializer,
-               root: 'sets', adapter: :json
+        page      = index_params[:page]&.to_i        || 1
```

Comment:
Do you still need the `||` values since you have defaults in `.search` methods?

```ruby
def self.search(page=1, per_page=10, attr_order=:updated_at, direction=:desc, search_term=nil)
```

Reply:
Yes, otherwise I would have to do 2 different calls: one when I've the value from the parameters  and when I don't. Or do you see another way? I could do `nil` check in the search method but it's basically the same thing.

Reply:
Consider passing index_params to `.search` method